Sources/Cache.swift

```diff
@@ -117,7 +117,7 @@ public final class Cache: Caching {
     }
 
     /// Removes all cached images.
-    public dynamic func removeAll() {
+    @objc public dynamic func removeAll() {
```

Comment:
I haven't yet went through all of Swift 4 changes. Not sure, why is this change necessary?